packages/ag-charts-enterprise/src/features/error-bar/errorBar.ts

```diff
@@ -242,6 +247,27 @@ export class ErrorBars extends _ModuleSupport.BaseModuleInstance implements _Mod
         return { xLowerKey, xUpperKey, xErrorsID, yLowerKey, yUpperKey, yErrorsID };
     }
 
+    private static getDatumKey(datum: ErrorBarNodeDatum, key: string | undefined, offset: number): number | undefined {
+        // Check if the user input datum has the error value for `key`:
+        if (key === undefined) {
+            return undefined;
+        }
+        const value: unknown = datum.datum[key];
+        if (value === undefined) {
+            return undefined;
+        }
+
+        // The datum has an error value for `key`. Validate this user input value:
+        if (typeof value !== 'number') {
+            _Util.Logger.warnOnce(
+                `Found [${key}] error value (${value}) of type ${typeof value}. Expected number type`
```

Comment:
Nit: I would not include the raw `value` in the message, as the deduplication effect of `warnOnce` is likely to be negated by different non-numeric numbers? (`typeof value` seems sufficient)